.github/workflows/tests.yaml

```diff
@@ -87,6 +87,22 @@ jobs:
     - name: Run tests
       run: make test.unit.pretty
 
+  test-conversion:
```

Comment:
👍 

Just a nit: we need to add it to `passed` job's `needs` so that we actually require it to pass for a PR to be allowed to merge.

This list is defined here: https://github.com/Kong/kubernetes-configuration/blob/a3bbc4e797977ce17c324a8665be651962b70055/.github/workflows/tests.yaml#L275-L286

Reply:
Nice catch!